src/Symfony/Component/Form/Extension/Validator/ViolationMapper/ViolationMapper.php

```diff
@@ -188,7 +186,7 @@ private function matchChild(FormInterface $form, PropertyPathIteratorInterface $
             }
 
             /** @var FormInterface $child */
-            foreach ($children as $key => $child) {
+            foreach ($children as $i => $child) {
```

Comment:
I have an intention that make the key of `$children` meaningless. (The name of `iterator_to_array`'s 2nd parameter is `$use_keys`)